protocol/metadata.go

```
@@ -2,24 +2,26 @@ package protocol
 import (
 	"context"
+	"crypto/x509"
 	"fmt"
 
 	"github.com/google/uuid"
 
 	"github.com/go-webauthn/webauthn/metadata"
 )
 
-func ValidateMetadata(ctx context.Context, aaguid uuid.UUID, mds metadata.Provider) (err error) {
+func ValidateMetadata(ctx context.Context, mds metadata.Provider, aaguid uuid.UUID, attestationType string, x5cs []any) (protoErr *Error) {
```

golangci-lint (1.62.2)

cyclomatic complexity 21 of func `ValidateMetadata` is high (> 15)

(gocyclo)